### PR TITLE
Add new HTML5 FAPI element: url

### PR DESCRIPTION
**Pull request by jenlampton**
PR for https://github.com/backdrop/backdrop-issues/issues/228

### CODE REVIEW

File: core/modules/simpletest/tests/form.test [hunk 1]
```diff
@@ -1769,3 +1772,48 @@ class HTMLIdTestCase extends DrupalWebTestCase {
...
   }
 }
+
+/**
+ * Tests url element.
+ */
+class FormUrlTestCase extends DrupalWebTestCase {
+  protected $profile = 'testing';
+
+  public static function getInfo() {
```
Comment (quicksketch) [inline core/modules/simpletest/tests/form.test]: The contents of the getInfo() method need to be moved to the simpletest.tests.info file.

**Comment by quicksketch** [comment]
This looks great other than the new tests need to be exposed to SimpleTest properly using Backdrop's *.tests.info files.

**Comment by jenlampton** [comment]
Alright, I just pushed the change that updated the test.

**Comment by jenlampton** [comment]
Fo reals this time. New review?

**Comment by quicksketch** [comment]
This looks great! Tested it out locally and the comment URL field is now showing up as a URL field. Sweet! With all new tests passing, this looks good to me Merged in.

**Pull request merged**